src/integer_mod_q/modulus_polynomial_ring_zq/from.rs

```diff
@@ -99,6 +167,126 @@ impl FromStr for ModulusPolynomialRingZq {
     }
 }
 
+#[cfg(test)]
```

Comment:
please also compare the entries of the polynomial, such that the first coefficient is 1/-1 and the last is 1